src/client/pages/OfferNew/Checkout/UpsellCard/Card.tsx

```diff
@@ -4,9 +4,9 @@ import { colorsV3 } from '@hedviginsurance/brand'
 
 import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
 
-import { Button } from '../../../components/buttons'
+import { Button } from '../../../../components/buttons'
```

Comment:
Don't we normally import like this:

```suggestion
import { Button } from 'components/buttons'
```